### DISCUSSION TITLE
[Feature] Auto tag external library items

### DISCUSSION THREAD

**Discussion by PJGitHub9** [immich-app/immich#13739] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I see the requests to auto add nested albums or create albums from external libraries, but another idea is to use the 'tag' feature to do this. 

For example if my folder structure is as followed:

2015 09 - Trip
_2015 09 - Trip\Stop A\Person A_
2015 09 - Trip\Stop A\Person B
2015 09 - Trip\Stop B\Person A

2015 10 - Car Show

2020 12 - Christmas

I want all the items in that root folder to get tagged with the folder names. So everything in the italicized folder for example will get the following tags assigned "2015 09 Trip" "Trip A" "Person A". 

Then I can filter by tags "Trip A" or "2015 09". Would be a great alternative to using albums. (since i can already use the folder function, and i prefer creating my own albums instead of auto generating based on folders)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4279/#1673

**Reply by PJGitHub9** [reply]
Those are albums and not using tags. Ive been following those threads for a while. My suggestion is to utilize tags specifically instead of albums
